sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -620,11 +620,34 @@ export function isServiceBusMessage(possible: unknown): possible is ServiceBusMe
   return isObjectWithProperties(possible, ["body"]);
 }
 
+export let _featureAmqpBodyTypeEnabled = false;
```

Comment:
Now that we have a getter, this need not be exported anymore.

Reply:
Done!